src/config/oranda_config/funding.rs

```diff
@@ -1,17 +1,64 @@
+use camino::Utf8PathBuf;
 use schemars::JsonSchema;
 use serde::Deserialize;
 
 use crate::config::{ApplyLayer, ApplyOptExt};
 use crate::data::funding::FundingType;
+use crate::errors::*;
 
 /// Config for displaying funding information on your page
 #[derive(Debug, Default, Deserialize, JsonSchema)]
 pub struct FundingConfig {
     pub preferred_funding: Option<FundingType>,
+    pub yml_path: Option<String>,
+    pub md_path: Option<String>,
 }
 
 impl ApplyLayer for FundingConfig {
     fn apply_layer(&mut self, layer: Self) {
         self.preferred_funding.apply_opt(layer.preferred_funding);
+        self.yml_path.apply_opt(layer.yml_path);
+        self.md_path.apply_opt(layer.md_path);
+    }
+}
+
+impl FundingConfig {
+    // see if we can find a FUNDING.yml, if so update the config path
+
+    /// If we have a FUNDING.yml file, try to find it. If we fail, we disable funding support.
+    pub fn find_paths(real_this: &mut Option<Self>) -> Result<()> {
```

Comment:
`real_this` is not immediately understandable as a name for me- maybe we can say UserConfig or something more descriptive about the origin of the data?